skaffold.yml

```diff
@@ -0,0 +1,62 @@
+apiVersion: skaffold/v1beta11
+kind: Config
+
+profiles:
+- name: development
+  activation:
+  - command: dev
+  patches:
+  - op: replace
+    path: /build/artifacts/0/docker/buildArgs/NODE_ENV
+    value: development
+  - op: replace
+    path: /build/artifacts/0/docker/target
+    value: development
+  - op: replace
+    path: /build/artifacts/1/docker/buildArgs/NODE_ENV
+    value: development
+  - op: replace
+    path: /build/artifacts/1/docker/target
+    value: development
+
+build:
+  artifacts:
+  - image: elm-lmp-app_web
+    context: .
+    docker:
+      dockerfile: Dockerfile
+      target: production
+      buildArgs:
+        NODE_ENV: production
+        PORT: 3000
+    sync:
+      manual:
+      - src: '**/*.js'
+        dest: .
+      - src: '**/*.njk'
+        dest: .
+
+  - image: elm-lmp-api_api
+    context: ../elm-api-boilerplate/
```

Comment:
Our meeting the other day about how to organise repositories was motivated by this exact issue. Rearranging the repos is outside the scope of this PR, which is simply getting the existing Kubernetes stack into Skaffold so we can develop the web app and API effectively.

We have identified separate work to pull the Kubernetes configuration out into a separate ELM Deployment repository and create Helm charts for each service. I think the deployment repo could easily include overrides for development so that we can run local builds of each service together. I wouldn't like to be in a position where we have to publish service changes to a container registry before we can test them working together.